packages/avocode-email-tagsinput/test/e2e/avocode-email-tagsinput.spec.js

```diff
@@ -0,0 +1,154 @@
+import puppeteer from 'puppeteer'
+import { expect } from 'chai'
+import {
+  URLRoot,
+  tagsInputFieldSelector,
+  getAnchorAndFocus,
+  addEmailTag,
+  getTagNodes, getInputText,
+} from './test-utils'
+describe('Avocode Email Tags Input', () => {
+  let browser
+  let page
+
+  beforeAll(async () => {
+    browser = await puppeteer.launch(browserOptions)
+  })
+
+  afterEach(async () => {
+    await page.close()
+  })
+
+  afterAll(async () => {
+    await browser.close()
+  })
+
+  describe('Basic Email Tags Input', () => {
+    beforeEach(async () => {
+      page = await browser.newPage()
+      await page.goto(URLRoot)
+    })
+
+    it('should render editor', async () => {
+      const editor = await page.waitForSelector(tagsInputFieldSelector)
+      expect(editor).to.exist
```

Comment:
I hate to be a strickler, but I'd kindly ask you to add new line between spec `expect` and rest of the code to make it consistent with other specs in `better-react-tagsinput`.

I should probably add linting rules to enforce this.

Reply:
Can you please add new line everywhere, so it looks like this?

```js
const editor = await page.waitForSelector(tagsInputFieldSelector)

expect(editor).to.exist 
```

Please make it analogically everywhere.